www/activities/settings/views/statistics.html

```diff
@@ -72,6 +72,7 @@
             </div>
           </div>
         </li>
+
```

Comment:
Please remove this empty line. This seems to be a leftover from an earlier change.

Reply:
done